tests/queries/0_stateless/03254_timeseries_functions_various_arguments.reference

```diff
@@ -27,6 +27,9 @@
 [0.85]
 [1]
 [8]
+[NULL,1,3,4,4,NULL,5,8]
+[NULL,1,3,4,4,4,5,8]
+[NULL,NULL,1,3,4,4,4,5,8]
```

Comment:
Without this fix these two lines were
```
[0,1,3,4,4,4,5,8]
[0,0,1,3,4,4,4,5,8]
```
which is incorrect. The input time series has values `[1, 1, 3, 4, 5, 5, 8, 12, 13]` and no value `0` at all!